docusaurus.config.js

```diff
@@ -1,293 +1,298 @@
+              },
+            ],
+          },
+          {
+            title: "Developer Resources",
+            items: [
+              {
+                label: "Status Page",
+                href: "https://status.console.cloud.mia-platform.eu"
+              },
+              {
+                label: "Guidelines",
+                to: "/docs/guidelines/git_vademecum",
+              },
+              {
+                label: "Library",
+                href: "https://resources.mia-platform.eu/en/library",
+              },
+              {
+                label: "GitHub",
+                href: "https://github.com/mia-platform",
+              },
+              {
+                label: "GitHub Marketplace",
+                href: "https://github.com/mia-platform-marketplace",
+              },
+            ],
+          },
+          {
+            title: "Education & Support",
+            items: [
+              {
+                label: 'Support',
+                href: 'https://makeitapp.atlassian.net/servicedesk/customer/portal/21'
+              },
+              {
+                label: 'FAQ',
+                to: '/docs/getting_started/faqs'
+              },
+              {
+                label: "Getting Started",
+                to: "/docs/overview/mia_platform_overview",
+              },
+            ],
+          },
+        ],
+        copyright: `Copyright © ${new Date().getFullYear()} Mia srl. All rights reserved. Built with Docusaurus.`,
+      }
     },
-    footer: {
-      style: "dark",
-      links: [
-        {
-          title: "Mia-Platform",
-          items: [
-            {
-              label: "How to install",
-              to: "/docs/info/how_to_install",
-            },
-            {
-              label: "Bug Policy",
-              to: "/docs/info/bug_policy",
-            },
-            {
-              label: "Supported browser",
-              to: "/docs/info/supported_browser",
-            },
-            {
-              label: "Open Source Software",
-              to: "/docs/info/oss",
-            },
-            {
-              label: "Subprocessor",
-              to: "/docs/info/subprocessor",
-            },
-            {
-              label: "Service Level Agreement",
-              to: "/docs/info/mia_service_level_agreement",
-            },
-            {
-              label: "Audit Process",
-              to: "/docs/info/audit_process",
-            },
-          ],
-        },
+    presets: [
+      [
+        "@docusaurus/preset-classic",
         {
-          title: "Company",
-          items: [
-            {
-              label: "Site",
-              href: "https://www.mia-platform.eu/en/",
-            },
-            {
-              label: "About",
-              href: "https://www.mia-platform.eu/en/company/about",
+          docs: {
+            remarkPlugins: [mdxMermaid.default, {
+              theme: {light: 'dark', dark: 'forest'}
+            }],
```

Comment:
```suggestion
            remarkPlugins: [mdxMermaid.default, {
              theme: {light: 'default', dark: 'dark'}
            }],
```
Despite the changes reported by GitHub, I only returned the config object in a function and I added this preset with the themes that seem not working.